releasenotes/notes/0.19/0.19-prelude-65c295aa9497ed48.yaml

```diff
@@ -0,0 +1,28 @@
+---
+prelude: |
+    The Qiskit Terra 0.19 release highlights are:
+
+    * A new version of the abstract Qiskit/hardware interface, in the form of
+      :class:`.BackendV2`, which comes with a new data structure
+      :class:`~.transpiler.Target` to allow backends to better model their
+      constraints for the :ref:`transpiler <qiskit-transpiler>`.
+
+    * An :ref:`extensible plugin interface <qiskit-transpiler-plugins>` to the
```

Comment:
Oh good call adding this here, I didn't think of that but it's a good one to highlight too.